### PR TITLE
add webmock and factorygirl methods to rails_helper

### PR DESCRIPTION
**Pull request by hiattp**
It seems like we should never allow third party requests in the test suite, so we should require webmock by default. ~Also, seems like including FactoryGirl methods makes sense? I always add this config.~

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -60,4 +61,7 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  # FactoryGirl Config
+  config.include FactoryGirl::Syntax::Methods
```
Comment (janklimo) [inline spec/rails_helper.rb]: @hiattp this one is already in https://github.com/lockstep/rails_new/blob/master/spec/support/factory_girl.rb

Comment (hiattp) [reply spec/rails_helper.rb]: Oh, thats weird. When I created a new app it complained that create wasn't defined (I had to reference fg explicitly). Have you confirmed whether this works?

Comment (janklimo) [reply spec/rails_helper.rb]: It does, as long as you `require 'rails_helper'` in your spec

Comment (hiattp) [reply spec/rails_helper.rb]: DOH! Always forget that line...

Comment (janklimo) [reply spec/rails_helper.rb]: @hiattp I did the exact same thing last week 😄

**Pull request merged**